openedx/features/marketplace/tests/view_tests.py

```diff
@@ -38,21 +38,21 @@ def setUp(self):
         }
 
     def test_get_marketplace_listing_page(self):
```

Comment:
how about we change the function name to `test_get_ communityhub_listing_page ` to keep it consistent? this goes for all other functions as well.

Reply:
@monofal now we are just changing the URL name , no change in any other files